core/store/migrate/migrations/0128_pipeline_runs_brin_to_btree.sql

```diff
@@ -0,0 +1,22 @@
+-- +goose Up
+	-- Migrate idx_pipeline_runs_created_at to BTREE
+    DROP INDEX IF EXISTS idx_pipeline_runs_created_at;
+    CREATE INDEX idx_pipeline_runs_created_at ON public.pipeline_runs USING BTREE (created_at);
```

Comment:
To clarify why this helps:  it seems that ORDER BY is able to make full use of a BTREE index but not a BRIN index, even though both of them work fine when it's just an ordinary WHERE filter.

The pipeline_runs table is probably the only table that's large enough for it to matter, so I would lean toward keeping as much the same as we can in case there are unexpected side effects.  There are some downsides of a BTREE index to think about.  The main one is size, it will take up more space on the disk.  It might also take longer to update as new rows are added, but I'm not sure if that would be significant enough to worry about.